bin/build.sh

```diff
@@ -23,7 +23,12 @@ else
     image_name="dbt-${adapter}"
 fi
 
+tag="ghcr.io/popsql/${image_name}:${version}"
+
 docker build \
     --build-arg REQUIREMENTS_FILE="${requirements_file}" \
-    --tag "ghcr.io/popsql/${image_name}:${version}" \
+    --build-arg DBT_VERSION="${version}" \
```

Comment:
Adding this was forgotten as part of f83c4f06c3c37a11d20daf5bc10ef0efdf9b89a0, adding it here as I was using this script to test the entrypoint.